extensions/openwsman/processors/SourceInitiatedSubscriptionListener.cpp

```diff
@@ -249,11 +249,11 @@ bool SourceInitiatedSubscriptionListener::loadState() {
 
 std::string SourceInitiatedSubscriptionListener::Handler::millisecondsToXsdDuration(int64_t milliseconds) {
   char buf[1024];
-  snprintf(buf, sizeof(buf), "PT%lld.%03lldS", milliseconds / 1000, milliseconds % 1000);
+  snprintf(buf, sizeof(buf), "PT%ld.%03ldS", milliseconds / 1000, milliseconds % 1000);
```

Comment:
Both the old and the new version are incorrect. The format specifier of `int64_t` is the `PRId64` macro from `<cinttypes>`.
```suggestion
  snprintf(buf, sizeof(buf), "PT%" PRId64 ".%03" PRId64 "S", milliseconds / 1000, milliseconds % 1000);
```

Reply:
Updated in 267cda83a77a5a6d05f9df714d0c923814351826